Tests/SendConsignments/SendMorningShipmentTest.php

```diff
@@ -157,9 +157,11 @@ public function testSendOneConsignment()
 
             $this->assertEquals(true, preg_match("#^https://api.myparcel.nl/pdfs#", $myParcelCollection->getLinkOfLabels()), 'Can\'t get link of PDF');
 
-            echo "\033[32mGenerated morning shipment label: \033[0m";
-            print_r($myParcelCollection->getLinkOfLabels());
-            echo "\n\033[0m";
+            if (!getenv('CI')) {
```

Comment:
Als je HasDebugLabels gebruikt, kan je dit vervangen met:

`$this->debugLinkOfLabels($myParcelCollection, 'morning shipment');`

Zie ook andere plekken

Reply:
Done